nilearn/datasets/description/localizer_first_level.rst

```diff
@@ -0,0 +1,34 @@
+localizer first level
```

Comment:
dataset hosted here: https://osf.io/v7hsw/?view_only=

but cannot find a whole lot more info about it